config/Scheduler/worker.js

```diff
@@ -55,3 +54,20 @@ Queue.process("deleteEvent", async (job, done) => {
 	await Promise.all(promises);
 	done();
 });
+
+Queue.process("sendGeneralEmailJob", async (job, done) => {
+	let { data } = job;
+	let log = `${data.mailType} email sent to ${data.email} at ${Date(
```

Comment:
general email have the same mailtype as 'others', right?

Reply:
yes:)